CONTRIBUTING.md

```diff
@@ -277,6 +277,24 @@ COVERAGE_PROCESS_START DEFAULT=/coverage.rc
 
 The CI/CD system is based on [Drone platform](https://drone.io/) and the server is  https://ci.vmware.run/. More information is found at our [CI.md](https://github.com/vmware/docker-volume-vsphere/blob/master/CI.md)
 
+## Releases for GA and post GA
+The following is the plan for releases for GA and post GA. Before GA, the release name format remains the same.
+
+### Type of Release:
+We plan to have two types of releases: edge release and stable release.
+
+edge release: will be released every month for users who want a drop of latest.
+
+stable release: will be release every 3 month for users who want to a stable version.
+
+### Nomenclature for release:
+
+The Nomenclature for release will follow the format ```major-version. minor-version.build-version.release-type```.
```

Comment:
Remove the redundant space after "major-version."

Reply:
Done